code/style.css

```diff
@@ -128,13 +129,28 @@ button {
   margin-right: 4px;
   font-size: 16px;
   line-height: 26px;
-  font-family: 'Montserrat';
+  font-family: "Montserrat";
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
 }
 
+.selectOptionDiv {
```

Comment:
According to Technigo code guidelines, CSS selectors should be written in `kebab-case`